src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbBase.java

```diff
@@ -129,7 +129,7 @@ protected MtAmazonDynamoDbContextProvider getMtContext() {
      * @param tableName Name of the table.
      * @return true if the given table name is a multitenant table associated with this instance, false otherwise.
      */
-    protected boolean isMtTable(String tableName) {
+    public boolean isMtTable(String tableName) {
```

Comment:
needed this sort of call from the test.... I explore trying to delete all tables between test runs, so a listTables() command doesn't return tables from other test runs, but didn't have much luck. Gave up trying to clean up other tests' tables, and tried filtering to just the tables this mt-dynamo client manages instead

Reply:
Are you trying to delete virtual or physical tables in your test?

Reply:
its physical tables left behind from other test cases that's the problem. when using raw `amazonDb.listTables()` to try running a scan against all tables, I'm getting plenty of physical tables leaked from other tests. I made this public, so I can filter all the physical tables from raw `AmazonDb.listTables()` to the set of mtTables that each given implementation of mt-dynamo (byTable, byIndex, byAccount) actually is using

Reply:
makes sense. `MtAmazonDynamoDbStreamsBaseTest#deleteMtTables` is doing that too, but it was probably already able to access that method, because it's in the same package.

Reply:
We could also consider implementing `listTables` with and without context, similar to what we're doing for scans and `listStreams` (though the latter currently only supports without context).